playbooks/roles/ocp-config/templates/install-config.yaml.j2

```diff
@@ -22,6 +22,7 @@ networking:
   - {{ service_network }}
 platform:
   none: {}
+fips: {{ fips_compliant }}
```

Comment:
What is `fips_compliant`? 
Please test your code.

Reply:
@yussufsh FIPS https://docs.openshift.com/container-platform/4.7/installing/installing-fips.html
This is must have feature for OCP which needs to be set during installation.

Reply:
I have no doubt that install-config.yaml does support fips.
My concern is with Ansible variable naming here. `fips_compliant` is read here and its `fips` in defaults, examples, README.